Sources/App/Models/User.swift

```diff
@@ -9,6 +9,10 @@ import FluentSQLite
 import Vapor
 import Authentication
 
+struct UserData: Content {
+    let id: Int?
```

Comment:
Will we be adding more fields in the future for the UserData Object?

Reply:
We might. I'm going to clean this up quick.

Reply:
```
struct UserData: Content {
    let username: String
}

final class User: SQLiteModel {
    var id: Int?
    var userData: UserData
    var password: String
    
    init(id: Int? = nil, username: String, password: String) {
        self.id = id
        self.userData = UserData(username: username)
        self.password = password
    }
}

- extension User {
-     var userData: UserData {
-         return UserData(id: id, username: username)
-     }
- }
```

Reply:
We'll be storing their password? Will we be hashing it ourselves?

Reply:
The password is already stored and hashed. The UserData object is used so we can send data about the user without sending their hashed password in the object.